vividus-plugin-rest-api/src/main/java/org/vividus/steps/api/HttpRequestSteps.java

```diff
@@ -241,32 +236,6 @@ public void setCustomRequestConfig(ExamplesTable configItems) throws ReflectiveO
         httpTestContext.putRequestConfig(requestConfigBuilder.build());
     }
 
-    /**
-     * Waits for a specified amount of time until HTTP response code is equal to what is expected.
-     * <p>
-     * <b>Actions performed:</b>
-     * </p>
-     * <ul>
-     * <li>Execute sub-steps</li>
-     * <li>Check if HTTP response code is equal to what is expected</li>
-     * </ul>
-     *
-     * @param responseCode for example 200, 404
-     * @param duration     Time duration to wait
-     * @param retryTimes   How many times request will be retried; duration/retryTimes=timeout between requests
-     * @param stepsToExecute Steps to execute at each wait iteration
-     */
-    @When("I wait for response code `$responseCode` for `$duration` duration retrying $retryTimes times"
-            + "$stepsToExecute")
-    public void waitForResponseCode(int responseCode, Duration duration, int retryTimes,
```

Comment:
Why it is moved?

Reply:
In the step we validate response codes, which is more suitable for the HttpResponseValidationSteps.class and this class has own method for response validate also